src/_common/form-vue/form.service.ts

```diff
@@ -91,6 +91,15 @@ export class BaseForm<T> extends Vue {
 	private customErrors: string[] = [];
 
 	get valid() {
+		return this._isValid();
+	}
+
+	/**
+	 * This exists so we have a backing field in case we want to override valid()
+	 * and want to extend its function body.
+	 * Since you cannot invoke super getters directly, call this function instead of `super.valid`.
+	 */
+	protected _isValid() {
```

Comment:
im on the fence with this but @hworld will probably be more opinionated.

I think we should just mutate customErrors if we have any additional logic that needs to be handled by the form.